setup-slave.sh

```diff
@@ -45,6 +45,21 @@ if [[ $instance_type == r3* ]]; then
   fi
 fi
 
+# Work around for hi1.4xlarge instances without pre-formatted ext3 disks
+if [[ $instance_type == hi1.4xlarge ]]; then
```

Comment:
Of course. 
Only one thing I'm not really sure - all the time second disk looked like `/dev/xvdf`. Will it work if we will use `/dev/sdc` instead?